polars/lazy/dataframe.ts

```diff
@@ -369,7 +369,21 @@ export interface LazyDataFrame extends Serialize, GroupByOps<LazyGroupBy> {
    * @deprecated *since 0.13.0* use {@link unpivot}
    */
   melt(idVars: ColumnSelection, valueVars: ColumnSelection): LazyDataFrame;
-  unpivot(idVars: ColumnSelection, valueVars: ColumnSelection): LazyDataFrame;
+  /**
+   * @see {@link DataFrame.unpivot}
+   * @param options.streamable - Allow this node to run in the streaming engine.
```

Comment:
can you specify that this is `false` by default.

Reply:
I'm confused about the default value.

For dataframes, it's false and cannot be changed: https://github.com/pola-rs/polars/blob/28d81963c1c03d64f6a71396da61b1668664824d/py-polars/src/dataframe/general.rs#L377, which is fine for this change.

In lazyframes, it doesn't have a default? I can't really read rust though https://github.com/pola-rs/polars/blob/28d81963c1c03d64f6a71396da61b1668664824d/py-polars/src/lazyframe/mod.rs#L1088

in python, streamable for lazyframes is true by default https://docs.pola.rs/api/python/stable/reference/lazyframe/api/polars.LazyFrame.unpivot.html#polars.LazyFrame.unpivot

in nodejs-polars in dataframe.rs https://github.com/pola-rs/nodejs-polars/blob/main/src/lazy/dataframe.rs#L516 it's `streamable: streamable.unwrap_or(false),`, which I'm assuming means false by default.

Reply:
hmm, yeah I don't have a ton of context into this either. maybe @ritchie46 or @stinodego would have more context.